shared/domains/chelonia/utils.js

```diff
@@ -747,6 +749,7 @@ export function eventsAfter (contractID: string, sinceHeight: number, limit?: nu
           }
         }
       } catch (e) {
+        console.error('[eventsAfter] Error', { lastUrl }, e)
```

Comment:
Why log `{ lastURL }` instead of `lastURL`?

Reply:
`lastURL` isn't defined